vmck/migrations/0001_initial.py

```diff
@@ -1,7 +1,6 @@
-# Generated by Django 2.1.5 on 2019-02-14 00:31
+# Generated by Django 2.2.4 on 2019-08-14 14:45
```

Comment:
Wait why are you rewriting the initial migration? You don't normally change old migrations. I was asking you to merge the migrations *from your pull request*, because it makes no sense to have several migrations in a single patch.